src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1072,8 +1068,9 @@ public float CalculateTotalDigestionSpeed()
 
     public override void OnLightLevelChanged(float lightLevel)
     {
-        var maxLightLevel = Editor.CurrentPatch.GetCompoundAmount("sunlight", CompoundAmountType.Maximum);
-        var templateMaxLightLevel = Editor.CurrentPatch.GetCompoundAmount("sunlight", CompoundAmountType.Template);
+        var maxLightLevel = Editor.CurrentPatch.Biome.Compounds[sunlight].Ambient;
+        var minLightLevel = Editor.CurrentPatch.Biome.MinimumCompounds[sunlight].Ambient;
+        var templateMaxLightLevel = Editor.CurrentPatch.Biome.Compounds[sunlight].Ambient;
```

Comment:
Either max or the template light level variable is wrong here. Or at least that has changed. Is this change intentional?

Reply:
Yes, the previous method returns percentage, which is not suitable for what we want to do later in the code.

Reply:
Well the variable name has to change then as it is not related to the template. I guess logically the change is fine as well as the player will be more interested in seeing what the light level can be in the patch, not what the archetype biome can have as the light level.

Reply:
Realized what you're saying was `maxLightLevel` is using `Compounds` instead of `MaximumCompounds`... let me rectify my dumb mistake.

Reply:
Also fix the fact that the "template" max light level doesn't actually have the template light level... The template light level if still actually needed can be retrieved from the Patch but not the biome conditions.

Reply:
Alright, figured it out, everything should be good.